GUIDELINES.md

```diff
@@ -171,6 +171,32 @@ Example:
 }
 ```
 
+### Request Nullable/Optional properties
+
+#### When to use `null` in properties
+
+1. When you want to update a property and unassign its value.
+2. If the property is optional and you are creating it you might want to use `null` to avoid the `if`s burden.
+
+#### When not to use `null` in properties
+
+1. Whenever you are updating a property that is optional/nullable and has a value assigned you don't want to remove.
+
+#### Nullable vs Optional
+
+When you set a property to `null` your intention is to remove any assigned value to it.
+When a property is optional you don't need to send the key in your request. The value will remain unchanged.
+
+**Note:** `Nullable` applies to the value of the property and `Optional` applies to the key of that property in the payload.
+
+#### Note on Empty strings
+
+Accepting empty strings does not have any meaning. If the goal is to unassign the value of a property then you should use `null` instead of an empty string.
+
+#### Note on Collections
+
+When the property you are updating is a collection, you **MUST** send all elements you want to keep or modify otherwise they will be removed.
```

Comment:
```suggestion
When the property you are updating is a collection, you **MUST** send all elements you want to keep or modify, otherwise they will be removed.
```